### ISSUE TITLE
escaped backslashes in translated strings

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-ios#185]
quotes (single and double quotes) in the the translations seems to be prefixed by an unnecessary double-backslash, eg. 

```
"qrscan_ask_join_group" = "Do you want to join the group \\\"%1$@\\\"?"
```

this is one pair of backslashes too much and renders as 

```
Do you want to join the group \"groupname\"?
```

**Comment by OzancanKaratas** [comment]
If it doesn't have a bad appearance, we can use as is. So I wanted a test to look it.

**Comment by OzancanKaratas** [comment]
I believe I fixed this in pull request #190. iOS and Android translations was merged by @cyBerta. Please review and test the changes with @r10s.

**Comment by r10s** [comment]
cool, i'll have a look
